src/Google.Api.Gax.Grpc.Gcp/Google.Api.Gax.Grpc.Gcp.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\CommonProjectProperties.xml" />
+
+  <!-- For the moment, this package is versioned independently. -->
+  <PropertyGroup>
+    <Version>2.5.0-beta01</Version>
```

Comment:
beta00?
I assume this isn't going to be released right now?

Reply:
I was going to probably do so, unless we have a good reason not to? (We want to release a beta *very soon* anyway, so that we can use it in Bigtable.)